GameServerLib/Logic/ObjectManager.cs

```diff
@@ -380,5 +380,27 @@ public bool TeamHasVisionOn(TeamId team, GameObject o)
 
             return false;
         }
+
+        public int CountUnitsAttackingUnit(AttackableUnit target)
+        {
+            int count = 0;
+            try
+            {
+                var enemyUnits = GetObjects();
+                var team = target.Team == TeamId.TEAM_BLUE ? TeamId.TEAM_PURPLE : TeamId.TEAM_BLUE;
+                count = enemyUnits.Count(x =>
+                    x.Value is ObjAiBase aiBase &&
+                    aiBase.Team == team &&
+                    aiBase.IsDead == false &&
```

Comment:
not sure how I feel about comparing boolean variables to true/false. Why not just use 
`!aiBase.IsDead`